tests/networkpolicy_test.go

```diff
@@ -104,14 +94,18 @@ var _ = Describe("[rfe_id:150][crit:high][vendor:cnv-qe@redhat.com][level:compon
 
 		It("[test_id:1511] should be failed to reach vmia from vmib", func() {
 			By("Connect vmia from vmib")
-			ip := vmia.Status.Interfaces[0].IP
-			assertPingFail(ip, vmib)
+			Expect(vmia.Status.Interfaces[0].IPs).NotTo(BeEmpty())
+			for _, ip := range vmia.Status.Interfaces[0].IPs {
+				assertPingFail(ip, vmib)
+			}
```

Comment:
Please encapsulate this into a function, then use it.

Reply:
you mean to encap this right ?
```
			for _, ip := range vmia.Status.Interfaces[0].IPs {
				assertPingFail(ip, vmib)
			}
```

Reply:
done, if needed will squash the commit to other one